### PR TITLE
GLT-452 Added read-only fields to display detailed sample attributes.

### PR DESCRIPTION
**Pull request by tonydebat**
Added note to indicate that these attributes can only be modified in the
bulk edit form.

### CODE REVIEW

**Comment by apmasell** [comment]
:+1:

File: miso-web/src/main/webapp/pages/editSample.jsp [hunk 1]
```diff
@@ -280,6 +280,145 @@
       <td>Emptied:</td>
       <td><form:checkbox id="empty" path="empty"/></td>
     </tr>
+    
+    <c:if test="${!empty sample.sampleAdditionalInfo}">
+            <div class="bs-callout bs-callout-info">
+              <p>Some sample fields shown below cannot be updated on this page. Use the bulk edit form to modify these fields.</p>
+            </div>
```
Comment (callunity) [inline miso-web/src/main/webapp/pages/editSample.jsp]: Can we add a link to this page? 
`<a href='<c:url value="/miso/sample/bulk/edit/${sample.id}"/>/>Use the bulk edit form</a>`

Comment (tonydebat) [inline miso-web/src/main/webapp/pages/editSample.jsp]: Added link to bulk edit.

**Comment by callunity** [comment]
:+1:

**Pull request merged**